bin/ream/src/cli/mod.rs

```diff
@@ -33,6 +38,26 @@ pub struct NodeConfig {
     )]
     pub network: Arc<NetworkSpec>,
 
+    /// HTTP port number
+    #[arg(long, default_value_t = DEFAULT_HTTP_PORT)]
+    pub http_port: usize,
+
+    /// HTTP bind address
+    #[arg(long,default_value_t = DEFAULT_HTTP_ADDRESS.to_string())]
+    pub http_address: String,
```

Comment:
```suggestion
    pub http_address: IpAddr,
```

This should be an IpAddr